standards/code-style.adoc

```diff
@@ -325,19 +331,19 @@ class MyClass {
 
 Braces around statements are required.
```

Comment:
Do we want to add an exception here for single-line if statements? Kotlin forwent the ternary operator in favor of that syntax: 
`val result = if (foo) bar else baz`

Reply:
An exception seems warranted to me given the lack of a ternary. 

Google's guide makes a braces exception in the following two cases:

https://android.github.io/kotlin-guides/style.html#braces
"Braces are not required for when branches and if statement bodies which have no else if/else branches and which fit on a single line."

https://android.github.io/kotlin-guides/style.html#expressions
"An if/else conditional that is used as an expression may omit braces only if the entire expression fits on one line."

@dandroid6000 's example seems to refer to the second case, which I agree with.

On the first case, I'm conflicted but lean toward accepting that one also. If we don't have strong opinions on that in-house, it makes sense to bias toward Google's guide. Thoughts @jdrider @kentonw @fuentespatrick @PerkinsRyan @mwolfson ?

Reply:
I've made note of this item. We'll move it outside the scope of this PR but include it in a subsequent PR in the next few weeks.